vllm/distributed/device_communicators/shm_broadcast.py

```diff
@@ -214,6 +215,8 @@ def __init__(
             self.remote_socket = context.socket(XPUB)
             self.remote_socket.setsockopt(XPUB_VERBOSE, True)
             remote_subscribe_port = get_open_port()
+            if ':' in connect_ip:
```

Comment:
```suggestion
            if connect_ip.count(':') >= 4:
```

Reply:
I don't think that's quite right. `::1` is a valid IPv6 address.

Reply:
it could be `>=2`, but I don't think there's a valid reason for `:` to ever be in this string except for IPv6